src/ViewTrait.php

```diff
@@ -252,6 +253,28 @@ public function setParameter(string $id, $value): self
         return $this;
     }
 
+    /**
+     * Add values to end of common array parameter. If specified parameter does not exist or him is not array,
+     * then parameter will be added as empty array.
+     *
+     * @param string $id The unique identifier of the parameter.
+     * @param mixed ...$value Value(s) for add to end of array parameter.
+     *
+     * @return static
+     */
+    public function addToArrayParameter(string $id, ...$value): self
+    {
+        /** @var mixed $array */
+        $array = $this->parameters[$id] ?? [];
+        if (!is_array($array)) {
+            $array = [];
```

Comment:
I'd throw `\InvalidArgumentException` in this case.

Reply:
Done.